Source/CarthageKit/Project.swift

```diff
@@ -1392,7 +1392,7 @@ private func dSYMForFramework(_ frameworkURL: URL, inDirectoryURL directoryURL:
 				.flatMap(.merge) { dSYMURL in
 					return UUIDsForDSYM(dSYMURL)
 						.filter { (dSYMUUIDs: Set<UUID>) in
-							return dSYMUUIDs == frameworkUUIDs
+                            return !dSYMUUIDs.isDisjoint(with: frameworkUUIDs)
```

Comment:
Looks like the indentation is wrong (tabs vs spaces).

Reply:
Should be fixed now!